regressionTests.cmake

```diff
@@ -457,6 +457,13 @@ add_test_compareECLFiles(CASENAME multflt_model2
 			  REL_TOL ${rel_tol}
 			  DIR model2)
 
+add_test_compareECLFiles(CASENAME multflt_sched_model2
+			  FILENAME 3_A_MPI_MULTFLT_SCHED_MODEL2
+			  SIMULATOR flow
+			  ABS_TOL ${abs_tol}
+			  REL_TOL ${rel_tol}
+			  DIR model2)
+
```

Comment:
the new test ain't gonna magically be updated, you have to adjust update_reference_data.sh

Reply:
Thanks, I forgot.